### ISSUE TITLE
[SCREENREADER]: Direct Deposit - Routing number alert box is not announcing itself to assistive devices

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#1437]
## Description

JAWS and NVDA are not announcing the alertbox when I try to enter an invalid routing number and press the `Update` button. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

*As a screen reader user, I want to hear the alertbox read out when I enter an invalid routing number.

## Environment

* Windows 10
* IE11, Chrome (latest), Firefox (latest)
* JAWS, NVDA

## Possible Fixes (optional)

Adding a div with a `role` and `aria-atomic` attributes are recommended in [ARIA19: Using ARIA role=alert or Live Regions to Identify Errors](https://www.w3.org/TR/WCAG20-TECHS/ARIA19.html)

```html
<!-- This empty div should be in the modal on first render -->
<div id="errors" role="alert" aria-atomic="true">
  <!-- Now add the alert box when no valid routing number is found -->
  <div class="usa-alert usa-alert-error">
    <div class="usa-alert-body">
      <div class="usa-alert-text">
        <p>We couldn’t find a bank linked to this routing number. Please check your bank’s 9-digit routing number and enter it again.</p>
      </div>
    </div>
  </div>
</div>
```
![Screen Shot 2019-08-29 at 2 01 51 PM](https://user-images.githubusercontent.com/934879/63969763-2f456f80-ca68-11e9-9a18-249ff501edec.png)

## WCAG or Vendor Guidance (optional)

* [Error Identification: Understanding SC 3.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-identified.html)

**Comment by Justin-Pickett** [comment]
@tlei123 can you validate this in QA please. Thank you.

**Comment by tlei123** [comment]
> @tlei123 can you validate this in QA please. Thank you.

@mojaray2k ,
Please route* to @1Copenut for Validation once [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/10715)'s merged and change is on Staging \[the PR's not Merged yet due to some failing Jenkins checks].

Thanks much!

\*For a11y updates/bugfixes (anything involving assistive devices), @1Copenut’s the appropriate validator, because he has the specialized screenreader licenses which QA lacks.

cc @Justin-Pickett

**Comment by 1Copenut** [comment]
@Justin-Pickett I've tested this with VoiceOver and NVDA (Firefox, Chrome) and it's responding as expected so far. Going to test JAWS later today and close out ASAP.

**Comment by 1Copenut** [comment]
@Justin-Pickett Just finished testing with JAWS (IE11 and Chrome) and it worked as expected. Moving this issue to close.
